### PR TITLE
Update wording.

### PR DESCRIPTION
**Pull request by wenottingham**

### CODE REVIEW

File: awx/ui/client/src/license/license.partial.html [hunk 1]
```diff
@@ -127,26 +127,19 @@
 					</div>
 					<div class="form-group License-detailsGroup" ng-if="licenseMissing">
 						<span class="License-helperText">
-							<translate>By default, Tower collects and transmits analytics data on Tower usage to Red Hat.  You can uncheck these boxes to disable sending data to these services:</translate>
+							<translate>
+							By default, Tower collects and transmits analytics data on Tower usage to Red Hat.  There are two categories of data collected by Tower. For more information, see this <a target="_blank" href="http://docs.ansible.com/ansible-tower/latest/html/installandreference/user-data.html#index-0">this Tower documentation page</a>. Uncheck the following boxes to disable this feature.</translate>
```
Comment (ryanpetrello) [inline awx/ui/client/src/license/license.partial.html]: this this Tower documentation page

**Review by ryanpetrello — APPROVED** [review]
minor nit

File: awx/ui/client/src/license/license.partial.html (hunk 1 above)
Comment (jlmitch5) [reply awx/ui/client/src/license/license.partial.html]: Tab this in one

**Review by jlmitch5 — APPROVED** [review]
Wording looks good (aside from the one thing ryan mentioned and tabbing the ext in one).

**Comment by wenottingham** [comment]
updated

**Comment by ryanpetrello** [comment]
cc @tvo318 this will affect the display of the license screen

**Pull request merged**